src/main/webapp/script.js

```diff
@@ -311,6 +311,7 @@ function createQuestionElement(question, hasRedirect) {
 
   const iconElement = document.createElement('i');
   iconElement.setAttribute('id', 'icon' + question.id);
+  iconElement.setAttribute('style', 'cursor: pointer');
```

Comment:
If possible, change this to CSS.

Reply:
but since the icons are dynamically generated and each one has a unique id I think that would be inconvenient